### DISCUSSION TITLE
Multiple Snippets Libraries

### DISCUSSION THREAD

**Discussion by v-c0de** [massCodeIO/massCode#75] [Ideas]
What about the possibility to easily select between different snippet profiles/libraries?

Potential use cases: 
If working on a project with others, there may be shared/communal snippets and each and every user wants to add/amend to them
But I don't want to dilute my own snippets (ie, some of the codes may not be relevant, or just personal preference in formatting etc)

So instead of having to go through preferences and open the different storage every time, might be an easier way to select 'switch profile/library'?

**Comment by antonreshetov** [comment]
massCode is a single user application, so it makes no sense.

**Reply by AVTPJ** [reply]
I would like to add my vote for this too. We have a couple of coders and I personally use Google Drive to store my db.json.

I have a lot of snippets I'd like to share with colleagues (and for them to share with me) but I'm hesitant for all of us to use the same .db on GD as it might introduce sync conflicts. Would it not be fairly straightforward to have multiple .db's and show them in the tree view as separate sources? You could potentially enable/disable 'write' access to specific storage in the preferences to protect that .db.

**Reply by Paurian** [reply]
I am in the same situation as AVTPJ. I'm working with a whole group of jr. coders with a couple of sr. level coders. All using Google Drive. We want to be able to have multiple "db.json" files read into the program (from different directories - or different files in the same directory - I don't care) so that everyone can share what they've learned without stepping into sync conflicts.

I'm thinking this would be reasonable to implement ... there is one main "db.json" that is read and written to by the person running the massCode and multiple other "db.json" files that can be read into the interface as read-only from other sources.
